portal-impl/src/com/liferay/portal/security/permission/ResourceActionsImpl.java

```diff
@@ -818,16 +818,12 @@ private ResourceActionsBag _getPortletResourceActionsBag(String name) {
 			Set<String> groupDefaultActions =
 				resourceActionsBag.getGroupDefaultActions();
 
-			if (groupDefaultActions.isEmpty()) {
-				groupDefaultActions.add(ActionKeys.VIEW);
-			}
+			groupDefaultActions.add(ActionKeys.VIEW);
```

Comment:
@dantewang I did not include this change, with current code, it is not that easy to understand it is safe to do this. Please add it back in later pulls iif needed.

Thanks.
Tina.